core/tauri-runtime-wry/src/lib.rs

```diff
@@ -715,7 +715,15 @@ unsafe impl Send for WindowBuilderWrapper {}
 impl WindowBuilderBase for WindowBuilderWrapper {}
 impl WindowBuilder for WindowBuilderWrapper {
   fn new() -> Self {
-    Self::default().focused(true)
+    #[allow(unused_mut)]
+    let mut builder = Self::default().focused(true);
+
+    #[cfg(target_os = "macos")]
+    {
+      builder = builder.title_bar_style(TitleBarStyle::Visible);
```

Comment:
This sounds like a workaround, because it breaks if user choses another title bar style. Let's add a comment why this workaround exists and backlink to the issue. Also add a TODO to find a proper fix.

Reply:
Added